isthmus/src/main/java/io/substrait/isthmus/expression/FunctionMappings.java

```diff
@@ -35,7 +35,9 @@ public class FunctionMappings {
                 s(SqlStdOperatorTable.MINUS_DATE, "subtract"),
                 s(SqlStdOperatorTable.DATETIME_PLUS, "add"),
                 s(SqlStdOperatorTable.EXTRACT, "extract"),
-                s(SqlStdOperatorTable.LIKE))
+                s(SqlStdOperatorTable.LIKE),
+                s(SqlStdOperatorTable.SUBSTRING, "substring"),
+                s(SqlStdOperatorTable.CONCAT, "concat"))
```

Comment:
Rebase on main and do a force push instead of pulling this in here.

Reply:
Ok, done

Reply:
Still doesn’t look right. There should be a single commit. If you rebase your local branch on an updated main branch, squash the commits into a single commit, and force push back here, it should fix it up.